tests/server.rs

```diff
@@ -71,6 +71,7 @@ where
             .expect("new rt");
         let srv = {
             let hits_counter = hits_counter.clone();
+            #[allow(clippy::async_yields_async)]
```

Comment:
I haven't seen any Clippy errors from this file. Is this because Clippy ignores test files?

Reply:
The trick is apparently to run `cargo clippy --tests`. We could add that to the CI.

I saw them pop up in VS Code.

Reply:
Ah, I see. Do you mind including that in this PR?